e2e/tests/api_tests/multiIntermediateTrack/api_multi_spec_test.js

```diff
@@ -14,24 +13,44 @@ async function prepareClaim(api_spec, mpScenario) {
 
 Scenario('1v1 full defence Multi claim Specified @api-nonprod-specified', async ({api_spec}) => {
   const mpScenario = 'ONE_V_ONE';
-  await prepareClaim(api_spec, mpScenario, defense);
-  await api_spec.defendantResponse(config.defendantSolicitorUser, defense, mpScenario, 'AWAITING_APPLICANT_INTENTION', false, true, claimAmountMulti);
-  await api_spec.claimantResponse(config.applicantSolicitorUser, defense, mpScenario, 'JUDICIAL_REFERRAL', false, true);
+  await prepareClaim(api_spec, mpScenario);
+  await api_spec.defendantResponse(config.defendantSolicitorUser, 'FULL_DEFENCE', mpScenario, 'AWAITING_APPLICANT_INTENTION', false, true, claimAmountMulti);
+  await api_spec.claimantResponse(config.applicantSolicitorUser, 'FULL_DEFENCE', mpScenario, 'JUDICIAL_REFERRAL', false, true);
+});
+
+Scenario('1v1 FULL_ADMISSION Multi claim Specified', async ({api_spec}) => {
+  const mpScenario = 'ONE_V_ONE';
+  await prepareClaim(api_spec, mpScenario);
+  await api_spec.defendantResponse(config.defendantSolicitorUser, 'FULL_ADMISSION', mpScenario, 'AWAITING_APPLICANT_INTENTION', false, true, claimAmountMulti);
+  await api_spec.claimantResponse(config.applicantSolicitorUser, 'FULL_ADMISSION', mpScenario, 'AWAITING_APPLICANT_INTENTION', false, true);
+});
+
+Scenario('1v1 PART_ADMISSION Multi claim Specified', async ({api_spec}) => {
+  const mpScenario = 'ONE_V_ONE';
+  await prepareClaim(api_spec, mpScenario);
+  await api_spec.defendantResponse(config.defendantSolicitorUser, 'PART_ADMISSION', mpScenario, 'AWAITING_APPLICANT_INTENTION', false, true, claimAmountMulti);
+  await api_spec.claimantResponse(config.applicantSolicitorUser, 'PART_ADMISSION', mpScenario, 'AWAITING_APPLICANT_INTENTION', false, true);
+});
+
+Scenario('1v1 COUNTER_CLAIM Multi claim Specified', async ({api_spec}) => {
+  const mpScenario = 'ONE_V_ONE';
+  await prepareClaim(api_spec, mpScenario);
+  await api_spec.defendantResponse(config.defendantSolicitorUser, 'COUNTER_CLAIM', mpScenario, 'AWAITING_APPLICANT_INTENTION', false, true, claimAmountMulti);
 });
 
 Scenario('1v2 full defence Multi claim Specified Different Solicitor', async ({api_spec}) => {
   const mpScenario = 'ONE_V_TWO';
   await prepareClaim(api_spec, mpScenario);
-  await api_spec.defendantResponse(config.defendantSolicitorUser, 'FULL_DEFENCE1', 'ONE_V_ONE_DIF_SOL',
-    'AWAITING_RESPONDENT_ACKNOWLEDGEMENT', false, true, claimAmountMulti);
-  await api_spec.defendantResponse(config.secondDefendantSolicitorUser, 'FULL_DEFENCE2', 'ONE_V_ONE_DIF_SOL',
-    'AWAITING_APPLICANT_INTENTION', false, true, claimAmountMulti);
+  await api_spec.defendantResponse(config.defendantSolicitorUser, 'FULL_DEFENCE1', 'ONE_V_ONE_DIF_SOL', 'AWAITING_RESPONDENT_ACKNOWLEDGEMENT', false, true, claimAmountMulti);
```

Comment:
FULL_DEFENCE1 not sure if typo?

Reply:
It's so it use the scenario in steps_LRspec.js, functionally the same as FULL_DEFENCE, but aligns with other 1v2 diff sol tests

   ONE_V_ONE_DIF_SOL: {
      FULL_DEFENCE1: data.DEFENDANT_RESPONSE('FULL_DEFENCE'),
      FULL_DEFENCE1_PBAv3: data.DEFENDANT_RESPONSE('FULL_DEFENCE', 'CREATE_CLAIM_SPEC_AFTER_PAYMENT'),
      FULL_DEFENCE1_PBAv3_Mediation: data.DEFENDANT_RESPONSE_1v2_Mediation('FULL_DEFENCE', 'CREATE_CLAIM_SPEC_AFTER_PAYMENT'),
      FULL_DEFENCE1_PBAv3_MULTI_CLAIM: data.DEFENDANT_RESPONSE_MULTI_CLAIM('FULL_DEFENCE', 'CREATE_CLAIM_SPEC_AFTER_PAYMENT'),